modules/v1/workspaces/controllers/WorkspaceController.php

```diff
@@ -33,4 +32,51 @@ public function actionGetWorkspaces()
             'query' => $query,
         ]);
     }
+
+    /**
+     * Get all users for workspace invitation
+     *
+     * @return ActiveDataProvider
+     */
+    public function actionGetAllUser()
```

Comment:
@skavt Better naming would be `actionActiveUsers`. Also this endpoint has nothing to do with workspace. It should be in users module.